docs/source/api/v4/servers.rst

```diff
@@ -425,8 +438,13 @@ Response Structure
 :profileDesc:    A :ref:`profile-description` of the :term:`Profile` used by this server
 :profileId:      The :ref:`profile-id` the :term:`Profile` used by this server
 :revalPending:   A boolean value which, if ``true`` indicates that this server has pending content invalidation/revalidation
-:rack:           A string indicating "server rack" location
-:status:         The status of the server
+
+	.. note:: While not officially deprecated, this is based on the values corresponding to ``revalUpdateTime`` and ``revalApplyTime``. It is preferred to use the timestamp fields going forward as this will likely be deprecated in the future.:
```

Comment:
Are we really not getting rid of these? Why not? They're just cruft at this point, it's a major API version change so we should be good to make breaking changes.

Reply:
If you think the fields still have value and shouldn't be removed, then we definitely shouldn't be telling people not use them or deprecating them (or this "soft deprecation" notice thingy).

I don't personally see much value because it's repeating information already contained in the response, specifically <var>updPending</var> ⇔ <var>queueTime</var> > <var>udpatesAppliedTime</var> and similar for `revalPending`. I think most clients will wind up looking at the timestamps anyway, simply because that gives you strictly more information. But if there's legitimately a use-case where people would rather use these "duplicated-ish" fields than the new timestamps, then by all means, keep them.

Reply:
> then we definitely shouldn't be telling people not use them or deprecating them (or this "soft deprecation" notice thingy).

Yeah I'll remove that now. That was a poor assumption on my part when I was first working on this.

I do believe there is value with those flags. All we have really done is provide additional information on the structs that can be utilized by clients if they care to, but I believe we do want to maintain that control in ATC which those flags still provide.

Reply:
If we get rid of these fields I believe that will break the version of t3c that is currently in master and that we are running in production.  I know we "shouldn't be relying on 4.x yadda, yadda" but we are and we should try not to break that if we can avoid it.